### PR TITLE
feat: fetch all dbs in a project

### PR DESCRIPTION
**Pull request by MutegekiHenry**
What does this PR do?

    Adds endpoints to retrieve all databases in a project for a user and all dbs for an admin

    This can be tested at GET on /projects/{project_id}/databases

    The admin route to get databases is GET /databases

### CODE REVIEW

File: app/controllers/project_database.py [hunk 1]
```diff
@@ -102,6 +102,28 @@ def post(self, project_id):
...
+        """
+        """
+        database_schema = ProjectDatabaseSchema(many=True)
+
+        project = Project.get_by_id(project_id)
+        if not project:
+            return dict(status='fail', message=f'Project with id {project_id} not found'), 404
+
+        databases = ProjectDatabase.find_all(project_id=project_id)
+        print(databases)
```
Comment (Mubangizi) [inline app/controllers/project_database.py:116]: You need to remove this print statement

**Review by Mubangizi — COMMENTED** [review]
Great work done, just one minor thing to point out

File: app/controllers/project_database.py (hunk 1 above)
Comment (MutegekiHenry) [reply app/controllers/project_database.py]: I have rectified this.

**Review by Mubangizi — APPROVED** [review]
LGTM

**Pull request merged**